docs/cloud-native-security/environment-variable-capture.asciidoc

```diff
@@ -0,0 +1,30 @@
+[[environment-variable-capture]]
+
+== Environment variable capture
+
+.Requirements
+[sidebar]
+--
+* This feature requires {stack} version 8.6 or higher.
+* In {stack} version 8.6, this feature is only available for Linux.
+--
+
+You can configure an {agent} policy to capture up to five environment variables ("env vars"). Captured env vars are associated with process events and recorded in the `process.env_vars` field.
+
+NOTE: Env var names must be 63 characters or fewer, and env var values must be 1023 characters or fewer. Values outside these limits are silently ignored.
```

Comment:
Maybe combine these notes, since they're both about env var limitations (see my comment at the end of the file):
```suggestion
[NOTE]
--
* Env var names must be 63 characters or fewer, and env var values must be 1023 characters or fewer. Values outside these limits are silently ignored.

* Env var names are case sensitive in Linux.
--
```